### PR TITLE
put label column name back into splits

### PR DESCRIPTION
**Pull request by DolanTheMFWizard**
*Issue #, if available:*

*Description of changes:*
when splits are generated by generate_train_test_split() function the name of the label gets removed from the series.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1319-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1319/1/index.html

**Review by Innixma — APPROVED** [review]

**Comment by Innixma** [comment]
Looks good. Thanks for the contribution!
